components/rmrk/Profile/ProfileDetail.vue

```diff
@@ -836,6 +876,9 @@ export default class Profile extends mixins(
     if (this.activeTab === 'sales') {
       this.fetchSalesEventByCreator()
     }
+    if (this.activeTab === 'offers') {
+      this.fetchOfferEvents()
+    }
```

Comment:
let's try to make a function to reuse this stuff

      if (this.activeTab === 'history') {
        this.fetchCollectionEvents()
      }
      if (this.activeTab === 'sales') {
        this.fetchSalesEventByCreator()
      }
      if (this.activeTab === 'offers') {
        this.fetchOfferEvents()
      }